diff --git a/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js b/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js
index f0d58d7..e3d65e3 100644
--- a/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js
+++ b/node_modules/@contentlayer/core/dist/generation/generate-dotpkg.js
@@ -140,7 +140,7 @@ const makeDataExportFile = ({ docDef, documentIds, assertStatement, }) => {
 export { default as ${dataVariableName} } from './${idToFileName(documentId)}.json'${assertStatement}
 `;
     }
-    const makeVariableName = flow(idToFileName, (_) => camelCase(_, { stripRegexp: /[^A-Z0-9\_]/gi }));
+    const makeVariableName = flow(idToFileName, (_) => camelCase(_, { stripRegexp: /[^\u4e00-\u9fa5A-Z0-9\_]/gi }));
     const docImports = documentIds
         .map((_) => `import ${makeVariableName(_)} from './${idToFileName(_)}.json'${assertStatement}`)
         .join('\n');
diff --git a/node_modules/@contentlayer/core/src/generation/generate-dotpkg.ts b/node_modules/@contentlayer/core/src/generation/generate-dotpkg.ts
index 863dfad..8e19ccb 100644
--- a/node_modules/@contentlayer/core/src/generation/generate-dotpkg.ts
+++ b/node_modules/@contentlayer/core/src/generation/generate-dotpkg.ts
@@ -324,7 +324,7 @@ export { default as ${dataVariableName} } from './${idToFileName(documentId)}.js
 `
   }
 
-  const makeVariableName = flow(idToFileName, (_) => camelCase(_, { stripRegexp: /[^A-Z0-9\_]/gi }))
+  const makeVariableName = flow(idToFileName, (_) => camelCase(_, { stripRegexp: /[^\u4e00-\u9fa5A-Z0-9\_]/gi }))
 
   const docImports = documentIds
     .map((_) => `import ${makeVariableName(_)} from './${idToFileName(_)}.json'${assertStatement}`)
